src/pages/userPage/EditUserPage/EditUserPage.tsx

```diff
@@ -47,6 +47,7 @@ export default function () {
   const secondPatern = /^[a-zA-Zа-яА-ЯІіЄєЇїҐґ'"\(\).`]{0,50}((\s+|-)[a-zA-Zа-яА-ЯІіЄєЇїҐґ'"\(\).`]{0,50})*$/;
   const message = shouldContain("тільки літери");
   const [form] = Form.useForm();
+  const MAX_AGE = 100;
```

Comment:
Why didn`t use lover camel case?

Reply:
I guess it's const variable

Reply:
Because it's constant value

Reply:
Ok, it`s my mistake (didn`t know about this rule of constants naming)